src/node_http2.cc

```diff
@@ -992,7 +992,8 @@ int Http2Session::OnStreamClose(nghttp2_session* handle,
   MaybeLocal<Value> answer =
     stream->MakeCallback(env->http2session_on_stream_close_function(),
                           1, &arg);
-  if (answer.IsEmpty() || answer.ToLocalChecked()->IsFalse()) {
+  Local<Value> def = v8::False(env->isolate());
+  if (answer.IsEmpty() || answer.FromMaybe(def)->IsFalse()) {
```

Comment:
`def` will never be used because the `FromMaybe()` call would only be used when `answer` is known not to be empty.